integrations/streamer_test.go

```diff
@@ -7844,6 +7844,7 @@ stream
 		.info(lambda: "count" > 6.0)
 		.warn(lambda: "count" > 7.0)
 		.crit(lambda: "count" > 8.0)
+		.details('details')
```

Comment:
Why change the test here? I'd like to test that the default behavior is not changed. Meaning if details is not specified that the pageDuty handler remains the same.